blockindex/contractstaking/event_handler.go

```diff
@@ -434,7 +434,15 @@ func (eh *contractStakingEventHandler) handleTransferEvent(event eventParam) err
 		return err
 	}
 
-	eh.tokenOwner[tokenID.Uint64()] = to
```

Comment:
same issue on L507?

Reply:
yes, please do a complete revisit of all the handlers, see if there are other places need to cover

Reply:
only `handleTransferEvent` need to update